### ISSUE TITLE
LoadError during startup with Redmine trunk r21288

### ISSUE THREAD

**Issue by vividtone** [onozaty/redmine-view-customize#84]
## Summary

View Customize plugin cannot run with Redmine trunk r21288. With the plugin installed, the following error occurs during startup and Redmine cannot be started.

Probably this is due to https://www.redmine.org/issues/32938.

```
/path/to/gems/ruby/3.0.0/gems/zeitwerk-2.5.1/lib/zeitwerk/kernel.rb:35:in `require': cannot load such file -- view_customize/view_hook (LoadError)
```

## Description

You can reproduce the issue by running `bin/rails c`

## Environment

- View customize plugin version: 2.8.1
- Redmine version: trunk r21288
- Ruby version: 3.0.0p0
- Rails version: 6.1.4.1
- Installed other plugins

**Comment by onozaty** [comment]
Thank you for your report.
Please give me some time so I can check the problem.

Thanks.

**Comment by onozaty** [comment]
The following two issues with Zeitwerk support have been fixed.

*  Change to use `require File.expand_path` to replace `require_dependency`.
* Fixed a mismatch between module name and directory name in `ViewHook`.

It has been released as v3.0.0.
